search-sentiments-map/public/scripts/modal-script.js

```diff
@@ -98,3 +98,95 @@ function displayTopResultsForCurrentTrend(countryCode) {
   }
   resultElement.innerHTML += '<i>Last updated on ' + date + '</i>';
 }
+
+/**
+ * Sets trends under the top-trends modal tab for the selected country.
+ * @param {string} countryCode The two-letter code of the selected country.
+ */
+function setCountryTrends(countryCode) {
+  const topTrendsTab = document.getElementById('top-trends-tab');
+  topTrendsTab.innerHTML = '';
+  fetch('/country-trends/' + countryCode).then(countryTrends =>
+      countryTrends.json()).then(trends => {
+        if (trends.length === 0) {
+          topTrendsTab.innerText = 
+              'Trends are not available for the selected country.';
+        } else {
+          trends.forEach(trend => {
+            const trendHeader = document.createElement('h5');
+            trendHeader.innerText = trend.topic;
+            topTrendsTab.appendChild(trendHeader);
+          });
+        }
+      });
+}
```

Comment:
I am unsure why this function is here. What is the difference between this function and the one starting on line 36? They have the same name and parameters, but the code is slightly different. This almost seems like an older version of the function that should be deleted.

Reply:
Only difference seems to be the `h4`
@ntarn ?

Reply:
This is probly some issue with her merge. @ntarn Plz keep delete this and keep the one above